SANBOXED_BUILDS.md

```diff
@@ -0,0 +1,119 @@
+# Sandboxed builds
+
+For convenience, OpenAssetIO provides two different sandboxed build
+environments.
+
+## Docker
+
+A simple way to perform a from-source build is to use the
+[OpenAssetIO-build docker image.](https://github.com/OpenAssetIO/OpenAssetIO/pkgs/container/openassetio-build)
+This image contains all the dependencies required for building and
+testing OpenAssetIO, and functions as a fully configured build
+environment.
+
+> **Note**
+>
+> The `openassetio` build environment is almost identical to
```

Comment:
I've simplified this, and just referenced the ASWF container, rather than explaining what the VFX reference platform is, since they're unique but related concepts I think thats enough, people without any context can find it through the links.

[9f3aadd](https://github.com/OpenAssetIO/OpenAssetIO/pull/735/commits/9f3aadd694e895a0a3d3b995688aa36291f3e84c)

Reply:
Nice, like it.